test/JDBC/expected/latest__verification_cleanup__14_3__sys-check_constraints-vu-verify.out

```diff
@@ -4,7 +4,7 @@ GO
 nvarchar
 ((sck_date_col IS NOT NULL))
 ((column_a = 0))
-((column_b <> 'wrong'))
+((column_b <> 'wrong' COLLATE "default"))
```

Comment:
Can this constraint be re-created from SSMS?

Reply:
Yes, the DDL can be re-created with the same constraint.